_data/sidebars/zh_lb4_sidebar.yml

```diff
@@ -0,0 +1,820 @@
+## definition of `menuitem`===========
+# title: string
+# url: /doc/zh/lb4/string
+# output: [ enum{web, pdf} ] (expressed as comma separated string in YAML)
+# children?: [ menuitem ]
+
+## top level property=================
+# entries: [ menuitem ]
+
+title: LoopBack 4
+url: /doc/zh/lb4/index.html
+children:
```

Comment:
Since you also translated `index.html` here. Let's add `translated: true` here as well